stable/prometheus-operator/values.yaml

```diff
@@ -1184,6 +1184,7 @@ prometheusOperator:
   serviceAccount:
     create: true
     name: ""
+    annotation: {}
```

Comment:
One is using annotation, the other one annotations. Can we use the plural everywhere?

Reply:
Sorry for the delay. Has been fixed.

Reply:
Fixed.